lib/form-validation.js

```diff
@@ -6299,6 +6299,24 @@
     return options.message;
   };
 
+  validate.validators.uuid = function (value, options) {
+    var uuid, pattern;
+
+    if (value) {
+      uuid = value.toString();
+      pattern = /^([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})$/gi;
+      var match = pattern.exec(uuid);
+
+      if (!match) {
+        return options.message;
+      }
+
+      return;
+    }
```

Comment:
eerder iets als 
`if (!isValidUUID(value)) { return options.message}` 

en dus isValidUUID functie er uit trekken

Reply:
Ik heb me gebaseerd op de code bij rrn. Daar dan ook aanpassen? Aangezien het code is dat naar AIV moet gepushed worden leek het me wel interessant om dezelfde stijl aan te houden.

Reply:
Misschien heeft @coemans daar een mening over? Ik vind het andere leesbaarder maar als het in de lijn met de rest MOET zijn ...

Reply:
Stijl/voorkeur verschilt natuurlijk altijd van team tot team. Aangezien de andere code zo in productie staat, zou ik daar niet te veel van afwijken om de kans groter te maken dat de PR vlot geaccepteerd wordt. Zoals @Pieter-Beckers nu gedaan heeft.

Los van deze PR, kan een PR voorgesteld worden waarin we de refactor uitvoeren zoals @pascalriquier voorstelt, maar lijkt me in deze misschien een stap te ver.